internal/services/account_sponsorship_service.go

```diff
@@ -33,8 +33,8 @@ func (e ErrOperationNotAllowed) Error() string {
 
 const (
 	// Sufficient to cover three average ledger close time.
-	CreateAccountTxnTimeBounds             = 18
-	CreateAccountTxnTimeBoundsSafetyMargin = 12
+	CreateAccountTxnTimeBounds             = 60
+	CreateAccountTxnTimeBoundsSafetyMargin = 60
```

Comment:
👋  This change is a bad idea, because the network usually times out in less than 30s. If you use 60 seconds here, you'll add complexity top how you will handle some indeterminate responses.

Reply:
I'm sorry, this was only for testing. the correct one is 30 seconds.